ksp_plugin/plugin.cpp

```diff
@@ -198,8 +216,9 @@ void Plugin::EndInitialization() {
   }
 
   // Construct the ephemeris.
-  ephemeris_ = solar_system.MakeEphemeris(default_ephemeris_fitting_tolerance,
-                                          DefaultEphemerisParameters());
+  ephemeris_ = solar_system.MakeEphemeris(
+      default_ephemeris_fitting_tolerance,
+      ephemeris_parameters_.value_or(DefaultEphemerisParameters()));
```

Comment:
Instead of using an `optional` I would just initialize `ephemeris_parameters_` in the constructor of the `Plugin` like the other parameters.

Reply:
As discussed, that doesn't work.